app/editor/src/features/admin/report-templates/constants/defaultAVOverviewRazorTemplate.ts

```diff
@@ -6,36 +6,21 @@ export const defaultAVOverviewBodyRazorTemplate = `@inherits RazorEngineCore.Raz
 @using System
 @using System.Linq
 @using TNO.Entities
-
 <style>
-  .section {
-    padding-bottom: 1rem;
-  }
-  .section-header {
-    background-color: black;
-    color: white;
-    text-align: center;
-    font-weight: bold;
-
-  }
-  table {
-    border: solid 1px black;
-    width: 100%;
-  }
-  td {
-    border: solid 1px;
-    padding: 0 1rem;
-  }
-  td.head {
-    font-weight: bold;
-  }
-  .small {
-    text-align: center;
-    width: 50px;
-  }
+a { text-decoration:none; }
```

Comment:
I've copied the latest in DEV hoping that this is the one Bobbi was working on.  I've created a separate story noting that currently the two templates are not in sync.